src/main/java/spoon/support/compiler/jdt/ParentExiter.java

```diff
@@ -462,7 +462,6 @@ public void visitCtPackage(CtPackage ctPackage) {
 				ctPackage.getTypes().remove(child);
 			}
 			ctPackage.getTypes().add((CtType<?>) child);
-			this.jdtTreeBuilder.context.addCreatedType((CtType<?>) child);
```

Comment:
This method appends the given CtType to a list in BuilderContext and this list is never used.